egs2/swbd_da/asr1/conf/tuning/train_asr_transformer_postencoder.yaml

```diff
@@ -0,0 +1,34 @@
+# network architecture
+# encoder related
+encoder: transformer
+encoder_conf:
+    output_size: 256    # dimension of attention
+    attention_heads: 4
+    linear_units: 2048  # the number of units of position-wise feed forward
+    num_blocks: 12      # the number of encoder blocks
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    attention_dropout_rate: 0.0
+    input_layer: conv2d # encoder architecture type
+    normalize_before: true
+
+# postencoder
+postencoder: transformers
+postencoder_conf:
+    model_name_or_path: "bert-base-cased"
```

Comment:
What other models can we use?
Can we basically pick up a model from https://huggingface.co/models?filter=transformers?
If so, it would be better to add such information as a comment here.

Reply:
I've tried more models and added a comment about it. However, it required some special handling, so maybe this PR needs another review.